Source/WebCore/rendering/RenderTableCell.cpp

```diff
@@ -353,8 +353,10 @@ LayoutSize RenderTableCell::offsetFromContainer(RenderElement& container, const
     ASSERT(&container == this->container());
 
     LayoutSize offset = RenderBlockFlow::offsetFromContainer(container, point, offsetDependsOnPoint);
-    if (parent())
-        offset -= parentBox()->locationOffset();
+    if (parent()) {
+        RenderElement* containerOfRow = container.container();
+            offset -= parentBox()->offsetFromContainer(*containerOfRow, point);
```

Comment:
I don't think it's clear that containerOfRow is always non-null here.

Reply:
Nit: there is an extra indentation for "offset".

Reply:
Should I add "comment" or add "releaseNonNull()"? or something else? I can fix "nit". Thanks!

Reply:
No, what I mean is that this seem to be missing a nullptr check.